application/common/models/Mfa.php

```diff
@@ -76,7 +76,16 @@ public function loadData(string $rpOrigin = '')
     {
         $this->data = [];
         if ($this->verified === 1 && $this->scenario === User::SCENARIO_AUTHENTICATE) {
-            $this->data += $this->authInit($rpOrigin);
+            try {
+                $this->data += $this->authInit($rpOrigin);
+            } catch (\Exception $exception) {
+                \Yii::error([
+                    'action' => 'load ' . $this->type . ' MFA data',
+                    'status' => 'error',
+                    'error' => 'authInit call failed (so skipping it): ' . $exception->getMessage(),
```

Comment:
Very minor comment, but perhaps "authInit failed, bypassing this MFA option" might be a tad better.